pallets/loans/src/lib.rs

```diff
@@ -476,10 +479,20 @@ pub mod pallet {
                     && market.collateral_factor < Ratio::one(),
                 Error::<T>::InvalidFactor,
             );
+            ensure!(
+                market.loose_collateral_factor < Ratio::one()
+                    && market.loose_collateral_factor > market.collateral_factor,
```

Comment:
I think the liquidation_threshold can be greater than or equal to the collateral factor.
liquidation_threshold >= collateral_factor